src/process/processrights.ts

```diff
@@ -205,8 +205,8 @@ export function isPotentialRoleOwner(user: UserDetails, roleId: string, workspac
   return false;
 }
 
-function addIfLicenseAllows(owners: IPotentialRoleOwners, user: UserDetails): void {
-  if (user.extras.roXtra.HasEFormulareEditAccess) {
+function addIfLicenceAllows(owners: IPotentialRoleOwners, user: UserDetails): void {
```

Comment:
Intentional change? Licen**s**e is also fine I think.

Reply:
I changed it, because it is spelled licence in the roXtra api

Reply:
Ah ok